packages/components/atoms/f-filter-pill/.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```

Comment:
nit: This file needs a newline at the end, happy for you to add this in your next pr 👍

Reply:
thanks!